src/helpers/money.ts

```diff
@@ -7,7 +7,7 @@ export const convertToCents = dollars => {
 }
 
 export const formatMoney = cents => {
-  if (cents === undefined) {
+  if (cents === undefined || Number.isNaN(cents)) {
```

Comment:
this conditional does not fully protect you from bad data in the case that 'cents' is either `null` or a non-numeric string. a more robust check for this particular case might be as follows.
```suggestion
  if (!cents || !Number.isFinite(+cents)) {
```